File: 백준/Bronze/1546. 평균/평균.py
# BOJ-평균(1546)


N = int(input()) #시험 본 과목의 개수

score = list(map(int, input().split())) # 현재 성적
M = max(score) #성적의 최댓값

new_score = [] #새로운 성적을 받기위해 빈 리스트 초기화

for a in score: #점수를 a에넣어주고
    new_score.append(a/M *100) #append()함수를 이용해 새로운 점수를 추가한다.

result = sum(new_score)/N #sum()함수를 이용해 합을 구하고 평균을 구한다.
print(result)

File: 백준/Bronze/1546. 평균/평균.js
// 백준  : 평균
const fs = require('fs');
const input =  fs.readFileSync('/dev/stdin').toString().trim().split('\n');

const N = input[0];
const score = input[1].split(' ').map(Number);
const maxScore = Math.max(...score);
let sum = 0;

let newArr = score.map((el) => {
    return el / maxScore * 100;
});

for(let i = 0 ; i < N ; i++){
    sum+=newArr[i];
}

console.log(sum/N);